bash_scripts/get_hashed_strings_v3.sh

```diff
@@ -0,0 +1,4 @@
+# Wrapper bash script that calls src/get_hashed_strings.js script
+#!/bin/bash
+
+node src/get_hashed_strings.js --resource=$TRANSIFEX_RESOURCE --token=$TRANSIFEX_AUTH_TOKEN
```

Comment:
Where will these variable values be coming from?

Reply:
These will be environment variables. TRANSIFEX_RESOURCE will be exported via Makefile and the auth token in job setup script(https://github.com/edx/jenkins-job-dsl-internal/blob/master/jobs/tools-edx-jenkins.edx.org/createTranslationJobs.groovy).